test/unit/widgets/buttons.test.ts

```diff
@@ -179,6 +191,54 @@ describe('continueButton', () => {
   })
 })
 
+describe('createLinkDiv', () => {
+  const obj = namedNode('https://test.com/#name')
+  const options: RenderAsDivOptions = {
+    deleteFunction: () => {},
+    link: true
+  }
+
+  it('adds a div to the element provided', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children.length).toBeGreaterThan(0)
+  })
+  it('makes the element draggable', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.getAttribute('draggable')).toEqual('true')
+  })
+  it('adds the style....', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children[0].getAttribute('style')).toEqual(linkDivStyle)
+  })
+  it('adds the deleteFunction of .... deleteButton with Check', () => {
+    const options = {
+      deleteFunction: () => {}
+    }
+    createLinkDiv(dom, element, obj, options)
+  })
+  it('adds the link icon and link for the uri if link option is true', () => {
+    const options = {
+      link: true
+    }
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children[0].children[0].nodeName).toEqual('A')
+  })
+})
+
+describe('createNameDiv', () => {
+  const obj = namedNode('https://test.com/#name')
+
+  it('adds a div to the element with textContent equal to Name', () => {
+    createNameDiv(dom, element, 'Name', obj)
+    expect(element.children.length).toBeGreaterThan(0)
+    expect(element.children[0].textContent).toEqual('Name')
+  })
+
+  it.skip('uses the name from the obj if no title is given', () => {
```

Comment:
**issue** unfinished test
**question** are you going to finish the tests in this PR?
**suggestion** Consider converting it to a draft PR